style.css

```diff
@@ -0,0 +1,687 @@
+    border-radius: 8px;
+  }
+
+  .contact {
+    background-color: #6070ff;
+    border-top-left-radius: 80px;
+    background-image: url(./assets/contact-form-shapes-mobile.png);
+    background-size: 50%;
+    background-position: right;
+    background-repeat: no-repeat;
+  }
+
+  .contact-form {
+    margin: 0 24px;
+    padding-bottom: 8px;
+  }
+
+  .contact-form h2 {
+    font-family: 'Poppins' Arial, Helvetica, sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    color: #fff;
+    align-items: center;
+    text-align: center;
+    padding-top: 103px;
+  }
+
+  .contact-form ul {
+    list-style: none;
+    display: flex;
+    flex-direction: column;
+    margin: 0;
+    padding: 0;
+    gap: 20px;
+  }
+
+  .contact-form input {
+    border: 1px solid #cfd8dc;
+    border-radius: 8px;
+    display: flex;
+    min-width: 100%;
+    height: 48px;
+    margin: 0;
+    padding: 0;
+  }
+
+  #message {
+    height: 170px;
+    width: 100%;
+    border-radius: 8px;
+  }
+
+  ::placeholder {
+    padding: 15px 38px 15px 16px;
+    font-family: 'Poppins' Arial, Helvetica, sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 17px;
+    line-height: 20px;
+    color: #172b4d;
+  }
+
+  #message::placeholder {
+    position: absolute;
+    left: 16px;
+    right: 38px;
+    top: 15px;
+    bottom: 15px;
+    padding: 0;
+    font-size: 15px;
+    line-height: 24px;
+    color: #b3bac5;
+  }
+
+  .contact-form button {
+    font-family: 'Poppins' Arial, Helvetica, sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    padding: 12px 16px;
+    color: #6070ff;
+    border-radius: 8px;
+    letter-spacing: 0.03em;
+    border: none;
+    background-color: #fff;
+  }
+
+  .shape {
+    display: flex;
+    justify-content: center;
+  }
+
+  h2.title360-d,
+  li.year-d,
+  li.ruby-d,
+  li.full-d,
+  li.facebook-d,
+  li.lead-d,
+  li.uber-d,
+  #image-d,
+  .card-d { display: none; }
+}
+
+@media (min-width: 768px) {
+  body { background-color: #e5e5e5; }
+
+  header {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    position: fixed;
+    width: 100%;
+    top: 0;
+  }
+
+  header a { padding: 26px 0 26px 155px; }
+
+  header nav { display: none; }
+
+  header ul {
+    display: flex;
+    list-style: none;
+    gap: 32px;
+    padding-right: 142px;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+  }
+
+  .container {
+    /* margin: 0;
+    border-bottom-left-radius: 80px;
+    background-color: #fff; */
+    position: relative;
+    width: auto;
+    height: auto;
+
+    /* Neutrals/N10 */
+
+    background: #f7f7f9;
+  }
+
+  .headline {
+    /* display: flex;
+    background-image: url(./assets/desktop/desktopHeader.png);
+    padding: 288px 260px 255px 260px;
+    background-repeat: no-repeat;
+    background-size: contain;
+    gap: 12px; */
```

Comment:
[OPTIONAL]

- KIndly ensure you remove all commented codes from your codebase so that your code can be clean and readable